src/utils/Firebase/firebase.js

```diff
@@ -36,23 +31,153 @@ class Firebase {
 
   doSignOut = () => this.auth.signOut();
 
-  // EXAMPLE ENDPOINT
-  getLesson = () => this.db.collection("custom_lesson");
-
-  //Gets last mastered lesson of a given student.
-  getLastMastered = deploymentAccountId =>
+  //Gets the most recent lesson that a student has completed
+  getLastMasteredLesson = deploymentAccountId =>
     this.db
       .doc(`deployment_account/${deploymentAccountId}/`)
       .get()
-      .then(documentSnapshot => {
-        documentSnapshot.get("profile.glenLearn.lastMasteredLesson");
-      });
+      .then(deploymentDoc => {
```

Comment:
maybe rename this to deploymentAccountDoc to differentiate between deployments and deployment accounts